elements/pfe-styles/demo/typography.html

```diff
@@ -0,0 +1,256 @@
+            <div class="pfe-headline-beta">.pfe-headline-beta: We’re the world’s leading provider of  <em>open source solutions</em>, where the <strong>best ideas win.</strong></div>
+            <p> Scelerisque eu eget consectetur vestibulum ullamcorper eros odio convallis eget suspendisse consectetur tristique in cum felis suspendisse est.</p>
+            <div class="pfe-headline-gamma">.pfe-headline-gamma: We’re the world’s leading provider of <em>open source solutions</em>, where the <strong>best ideas win.</strong></div>
+            <p> Scelerisque eu eget consectetur vestibulum ullamcorper eros odio convallis eget suspendisse consectetur tristique in cum felis suspendisse est.</p>
+            <div class="pfe-headline-delta">.pfe-headline-delta: We’re the world’s leading provider of <em>open source solutions</em>, where the <strong>best ideas win.</strong></div>
+            <p> Scelerisque eu eget consectetur vestibulum ullamcorper eros odio convallis eget suspendisse consectetur tristique in cum felis suspendisse est.</p>
+            <div class="pfe-headline-epsilon">.pfe-headline-epsilon: We’re the world’s leading provider of <em>open source solutions</em>, where the <strong>best ideas win.</strong></div>
+            <p> Scelerisque eu eget consectetur vestibulum ullamcorper eros odio convallis eget suspendisse consectetur tristique in cum felis suspendisse est.</p>
+            <div class="pfe-headline-zeta">.pfe-headline-zeta: We’re the world’s leading provider of <em>open source solutions</em>, where the <strong>best ideas win.</strong></div>
+            <p> Scelerisque eu eget consectetur vestibulum ullamcorper eros odio convallis eget suspendisse consectetur tristique in cum felis suspendisse est.</p>
+            <p class="pfe-text-lg">.pfe-text-lg: At vero eos et accusamus et iusto odio dignissimos ducimus <em>qui blanditiis praesentium</em> voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi <strong>sint occaecati cupiditate</strong> non provident, optio cumque nihil impedit quo minus id quod maxime placeat facere possimus, omnis voluptas assumenda est, omnis dolor repellendus.</p>
+           <p class="pfe-text">.pfe-text: We’re the <a href="https://foo.com">world’s leading provider</a> of <em>open source solutions</em>, where the <strong>best ideas win.</strong></p>
+            <p class="pfe-text-sm">.pfe-text-sm: We’re the world’s leading provider of <em>open source solutions</em>, where the <strong>best ideas win.</strong></p>
+            <p class="pfe-text-xs">.pfe-text-xs: We’re the world’s leading provider of <em>open source solutions</em>, where the <strong>best ideas win.</strong></p>
+          </pfe-card> 
+          </div>
+
+        </article>
+  </body>
+
+  <script>
+    var containers = document.querySelectorAll(".context-container");
```

Comment:
Are you using this JS on this page?  If not you can delete it.

Reply:
deleted